### ISSUE TITLE
/rest/db/file (and some other db request) don't works for paused folder

### ISSUE THREAD

**Issue by nynjardin** [syncthing/syncthing#6842]
Hi,

I use Syncthing v**1.7.1**

I got an error 404 when I try to get /rest/db/file from a file in a paused folder. To reproduce, easy, try to get data of a file in a paused folder and you'll get a 404 error about the request.

**Comment by nynjardin** [comment]
Hi!No news about this issue? or there is an alternative solution to know if a file have been updated when the folder is paused? Thanks

**Comment by AudriusButkevicius** [comment]
Nope, not really, and I don't think an older version would change anything.

**Comment by nynjardin** [comment]
> Nope, not really, and I don't think an older version would change anything.

Ho, ok, I have understood that it is the case on an older version, that why I ask!

So, any idea when it will be possible? I'm starting to create a database solution just to store completion of folder for each client... but if it's planned to be implemented in (a more or less near) futur, I can wait instead of build this database (I know it's "too much" for what I need, but I don't see any other "simple" solution)

Same to know if a folder have been updated on a server, without having to unpause the folder

Thank you :)

**Comment by AudriusButkevicius** [comment]
I don't think we plan to implement this

**Comment by nynjardin** [comment]
Alright, I'll start my solution, thanks for the answer :)
